doc/design/OrionHDFSSink.md

```diff
@@ -1 +1,139 @@
+        * `recvTimeTs`: UTC timestamp expressed in miliseconds.
+        * `recvTime`: UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)).
+        * `entityId`: Notified entity identifier.
+        * `entityType`: Notified entity type.
+        * `attrName`: Notified attribute name.
+        * `attrType`: Notified attribute type.
+        * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be Json object or Json array.
+        * `attrMd`: It contains a string serialization of the metadata array for the attribute in Json (if the attribute hasn't metadata, an empty array `[]` is inserted).
+    * `column`: A single Json line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and other for the metadata, called `<attrName>_md`), plus an additional field about the reception time of the data (`recvTime`).
+
+####Important notes regarding the persistence mode
+Please observe not always the same number of attributes is notified; this depends on the subscription made to the NGSI-like sender. This is not a problem for the `row` persistence mode, since fixed 8-fields Json documents are appended for each notified attribute. Nevertheless, the `column` mode may be affected by several Json documents of different lengths (in term of fields). Thus, the `column` mode is only recommended if your subscription is designed for always sending the same attributes, event if they were not updated since the last notification.
+
+###Hive
+A special feature regarding HDFS persisted data is the posssibility to exploit it through Hive, a SQL-like querying system. `OrionHDFSSink` automatically [creates a Hive external table](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Create/Drop/TruncateTable) (similar to a SQL table) for each persisted entity in the default database, being the name for such tables as `<username>_<fiware-service>_<fiware-servicePath>_<destination>_[row|column]`.
+
+The fields regarding each data row match the fields of the Json documents appended to the HDFS files. They are deserialized by using a [Json serde](https://github.com/rcongiu/Hive-JSON-Serde).
+
+###Example
+Assuming the following Flume event is created from a notified NGSI context data (the code below is an <i>object representation</i>, not any real data format):
+
+    flume-event={
+        headers={
+	        content-type=application/json,
+	        fiware-service=vehicles,
+	        fiware-servicepath=4wheels,
+	        timestamp=1429535775,
+	        transactionId=1429535775-308-0000000000,
+	        ttl=10,
+	        destination=car1_car
+        },
+        body={
+	        entityId=car1,
+	        entityType=car,
+	        attributes=[
+	            {
+	                attrName=speed,
+	                attrType=kmh,
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/5b6a45eb7c12d4db2e56ac0147eb6851b51c3d9